clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,153 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 5}}
```

Comment:
saying that 5 is large is a bit of exaggeration :)

Don't we want to test larger services?

Reply:
Good catch. these were 300, 150 and 10 respectively. I was testing locally with these smaller numbers to make sure the test runs. I committed them by mistake. Will fix.

Reply:
done